src/App.jsx

```diff
@@ -48,8 +46,8 @@ function App () {
   }
 
   // On finish functions for electron, Mturk
-  const defaultFinishFunction = () => {
-    jsPsych.data.get().localSave('csv', 'neuro-task.csv')
+  const defaultFinishFunction = (data) => {
+    data.localSave('csv', 'neuro-task.csv')
```

Comment:
I'm attempting to leave `jspsych` scoped to within the JsPsychExperiment component.  Fortunately, in this case we can accept the same `jsPsych.data.get()` as a callback parameter instead of explicitly accessing jspsych.